Apps/Src/ReadSwitches.c

```diff
@@ -1,146 +0,0 @@
-/* Copyright (c) 2022 UT Longhorn Racing Solar */
```

Comment:
Deleting this file is fine, but not without an alternative in place. While the new minion driver does make polling the switches completely unnecessary, we still need something to gate the enable line of the contactors. This logic *could* be moved into the initialization task that once it finishes up just hangs in a loop however.

Reply:
If you don't want to do it that way however, then, this task will need to remain, simply to poll the ignition switch and enable/disable contactors. Up to whoever is implementing it.